devs/run-tests/24arraymap.ts

```diff
@@ -152,6 +152,56 @@ function testArrayAt() {
     assert(str.at(-6) === undefined, "arrayAtUndefined")
 }
 
+function localeCompareHelper(a: string, b: string) {
+    let i = 0
```

Comment:
this is exactly what `<` and `>` do on strings, so the function can be simplified to:

```ts
if (a < b) return -1;
else if (a > b) return 1;
else return 0;
```

Reply:
Changed the implementation of `localeCompareHelper`